integration/test/version

```diff
@@ -0,0 +1 @@
+Directory Transport Version: 1.1
```

Comment:
@Jamstah Is this file necessary? I can’t see any user.

If it is, please move it into `integration/fixtures`, as `integration/fixtures/some-descriptive-name`.

Reply:
Sorry, that's a file created by running the integration tests. Should probably be added to .gitignore.